app/Main.hs

```diff
@@ -153,6 +153,16 @@ main = do
                 let dpath = rpath </> destPath
                 (Hap.exec . Hap.MkDir . parent) dpath
                 Hap.scpDir srcPath dpath
+              forM_ configLinkedFiles $ \fileToLink -> do
```

Comment:
@CristhianMotoche could you please extract this code out to a separate function and add some tests: